unit-tests/dds/d405.py

```diff
@@ -41,142 +42,142 @@ def colored_infrared_stream_profiles():
         # dds.video_stream_profile( 30, dds.stream_format("BGRA"), 1280, 720 ),
         # dds.video_stream_profile( 30, dds.stream_format("RGBA"), 1280, 720 ),
         # dds.video_stream_profile( 30, dds.stream_format("RGB2"), 1280, 720 ),
-        # dds.video_stream_profile( 30, dds.stream_format("rgb8"), 1280, 720 ),
+        dds.video_stream_profile( 30, dds.stream_format("rgb8"), 1280, 720 ),
```

Comment:
Why re-enable rgb8?

Reply:
I don't know if your question is for the colored infrared stream or general.

For the D405 color stream and also for the other modules I re-enabled rgb8 because that is what the users will see after LibRS will convert the received streams.
For the colored infrared it was more of inertia, doing the same all over the files. The stream is not broadcasted anyway.

Reply:
Fixed, as previously discussed